AndroidXCI/lib/src/main/kotlin/dev/androidx/ci/testRunner/vo/ApkInfo.kt

```diff
@@ -29,7 +29,7 @@ data class ApkInfo constructor(
     /**
      * A hash value computed from the contents of the ApkFile. Is good enough to be used as a unique identifier.
      */
-    val idHash: String
+    val idHash: String = ""
```

Comment:
i don't think we want to make these optional.

Reply:
done